Typescript_React/src/TodoList.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { TodoListItem } from './TodoListItem';
+import { Todo, ToggleTodo } from './Types';
+
+interface TodoListProps{
+    todos:Array<Todo>,
+    toggleTodo:ToggleTodo
+}
+
+export const TodoList:React.FC<TodoListProps> = ({ todos,toggleTodo}) =>{
+     return <ul>{todos.map(todo =>{
+        return <TodoListItem key={todo.text} todo={todo} toggleTodo={toggleTodo}/>
```

Comment:
No need to use {} and return keyword in an arrow function when function definition has only one line.